docs/tutorials/ci-cd.md

```diff
@@ -45,19 +50,16 @@ jobs:
 
 ```
 
-1. You are looking at an example project, your project will probably look a
-   little different. Important parts of how to integrate Flox with Github
-   Actions are highlighted bellow.
-2. `flox/install-flox-action` will install latest version Flox.
-3. `flox/activate-action` allows you to run a command inside the Flox
-   environment.
+1. You are looking at an example project, your project will probably look a little different. Important parts of how to integrate Flox with Github Actions are highlighted below.
+2. `flox/install-flox-action` will install the latest version of Flox.
+3. `flox/activate-action` allows you to run a command inside the Flox environment.
 
 ## CircleCI
 
-There is a [Flox Orb](https://github.com/flox/flox-orb) that can help you use
-Flox inside CircleCI.
+There is a [Flox Orb](https://github.com/flox/flox-orb) that helps you use Flox inside CircleCI.
+Similar to GitHub Actions there is a `flox/install` step and a separate `flox/activate` step.
```

Comment:
If a person is coming here to work on CircleCI, they may not have context for GHA, so it's probably not a helpful mention here.